src/components/bookActions.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook, removeBook } from '../redux/books/booksSlice';
+
+const BookActions = () => {
+  const dispatch = useDispatch();
+
+  const handleAddBook = () => {
+    const newBook = {
+      item_id: 'item4', // Generate a unique ID
+      title: 'New Book',
+      author: 'New Author',
+      category: 'Fiction',
+    };
+    dispatch(addBook(newBook)); // Dispatch the addBook action
+  };
```

Comment:
- [ ] I cannot add a book. I don't think the book addition feature is meant to be coded here. They should be done in the bookForm component. Kindly ensure books can be added.

- [ ] The remove book button should be on the book component itself, not beside the add button. That is, each book should have its own remove button.

  ![image](https://github.com/mershark/bookstore/assets/81468675/a9d2d8ff-dfba-44a3-aa28-1ba4b85391e0)